README.md

```diff
@@ -1,2 +1,29 @@
 # recover-s3-synindex
-To index S3 objects in Synapse from a S3 bucket attached to a Synapse project/folder 
+
+## Purpose
+This repository helps index S3 objects in Synapse from a S3 bucket attached to a Synapse project/folder. The code is written in R and run from a Rstudio EC2 instance.
+
+## Installation/Pre-requisites
+1. Spin up a new EC2 Rstudio Notebook instance.
+2. In the terminal, upgrade pip `python -m pip install --upgrade pip` before trying to install synapseclient.
+3. Install synapseclient in the terminal using the folloiwing command
+`pip install synapseclient`. 
+
+Note: If you are having issues during installation, consider upgrading your pip `python -m pip install --upgrade pip` before trying to reinstall synapseclient - this should clear most issues. If you still have issues try force re-install synapseclient from the terminal `pip install --upgrade --force-reinstall synapseclient`
+
+4. Create a `.Renviron` file in the home folder with the environment variables `SYNAPSE_USERNAME`, `SYNAPSE_PASSWORD` and `SYNASPECLIENT_INSTALL_PATH`, where `SYNAPSECLIENT_INSTALL_PATH` is the path where the script synapse is installed when synapseclient was installed in step 2.
```

Comment:
- Recommend using a synapse auth_token env variable in place of username and password
- Mention that you can get the install path for synapseclient using `pip show synapseclient` in case the user is not already aware of this

Reply:
changed login to via auth token and updated the readme to reflect the same. Also included `pip show synapseclient` in the readme (didn't know about this thanks Pranav)